### PR TITLE
New blog post on DSC Logging

### PR DESCRIPTION
**Pull request by ykuijs**
First PR to gather some feedback on the blog post

I have also updated the style.css to make sure ordered and unordered lists are rendered correctly.

---
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/dsccommunity.org/61)

### CODE REVIEW

**Comment by ykuijs** [comment]
@gaelcolas Are you up for a first review?

**Comment by gaelcolas** [comment]
I'll have a look when I get a chance. Weekend pretty busy, but if I get a chance I might review while traveling.

File: themes/dot/assets/css/style.css [hunk 1]
```diff
@@ -134,7 +134,7 @@ body {
 
 ol,
 ul {
-  list-style-type: none;
+  /* list-style-type: none; */
```
Comment (gaelcolas) [inline themes/dot/assets/css/style.css]: This would affect other parts of the site, so is best to submit in separate PR.

File: content/blog/how-to-use-dsc-logging.en.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Reading the active log file
+
+While diving into this topic, DSC presented me with a not so nice surprise:
+When a DSC job is running, not only the Get-DscConfigurationStatus cmdlet does
+not work, also the logfile that is currently being created cannot be read!
+Neither via Windows Explorer, nor via PowerShell.
+So how can you keep track of the progress of an active DSC job?
+
+Fortunately some digging around on the Internet resulted in a solution!
```
Comment (gaelcolas) [inline content/blog/how-to-use-dsc-logging.en.md]: Maybe cite your sources?

File: content/blog/how-to-use-dsc-logging.en.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+---
+title: "How to Use Dsc Logging"
+date: 2019-09-24T16:37:42+02:00
+type: "post"
+draft: false
```
Comment (gaelcolas) [inline content/blog/how-to-use-dsc-logging.en.md]: Please add `author: authorhandle` metadata.
We'll use it when I get a chance to display a card of the author on the article.

File: content/blog/how-to-use-dsc-logging.en.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+---
+
+PowerShell Desired State Configuration (DSC) logs its deployment progress and errors
```
Comment (gaelcolas) [inline content/blog/how-to-use-dsc-logging.en.md]: I'd specify that's the LCM, because technically that changes with PS7's Invoke-DscResource.

File: content/blog/how-to-use-dsc-logging.en.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+Success    26-9-2019 15:12:45        Consistency     PUSH  False                1
+Success    26-9-2019 15:12:44        Consistency     PUSH  False                1
+Success    26-9-2019 14:57:44        Consistency     PUSH  False                1
+Success    26-9-2019 14:42:45        Consistency     PUSH  False                1
+Success    26-9-2019 14:42:44        Consistency     PUSH  False                1
+Success    26-9-2019 14:27:44        Consistency     PUSH  False                1
+...
+```
+
+You can use the pipeline and filtering to extract the job you are interested in.
```
Comment (gaelcolas) [inline content/blog/how-to-use-dsc-logging.en.md]: `use the pipeline and filter`

File: content/blog/how-to-use-dsc-logging.en.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+(even when -Verbose hasn't been specified), all Verbose output is always written
+to log files in: **C:\Windows\System32\Configuration\ConfigurationStatus**
+
+For each job, DSC creates two files:
+
+1. **{\<DSC_Job_Id\>}-0.mof**
+   * This file contains LCM configuration, resources not in the desired state and
+     the results of the DSC job.
+1. **{\<DSC_Job_Id\>}-0.details.json**
+   * This file contains the verbose logging in JSON format.
```
Comment (gaelcolas) [inline content/blog/how-to-use-dsc-logging.en.md]: I think the numbering went wrong in your last edit

Comment (gaelcolas) [reply content/blog/how-to-use-dsc-logging.en.md]: @ykuijs You may not have seen this comment ☝️ 
Otherwise it looks good.

Comment (ykuijs) [reply content/blog/how-to-use-dsc-logging.en.md]: Numbering in Markdown is always 1. Markdown itself will increment where required.

The built in Markdown validator in VSCode pointed out that the original numbering was incorrect. That is why I corrected it. It should number just fine on the site.

Comment (gaelcolas) [reply content/blog/how-to-use-dsc-logging.en.md]: It's usually an indentation problem when it's flagged.
No Biggie, no need to hold for that.
I'll merge if your happy with it.
Ok to release?

**Comment by ykuijs** [comment]
Yes, go ahead!
